assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -824,8 +824,9 @@ const baseSelectors = {
 		invariant( moduleSlug, 'slug is required.' );
 		const modules = select( STORE_NAME ).getModules();
 
-		if ( ! modules ) {
-			return null;
+		// Return `undefined` if modules haven't been loaded yet.
+		if ( modules === undefined ) {
+			return undefined;
```

Comment:
As per Evan's comment https://github.com/google/site-kit-wp/issues/3226#issuecomment-834628021

Just to note, this file has several different ways of checking modules for being `null` or `undefined` or falsey, and then returns different things :see_no_evil: